### PR TITLE
change the registry value

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5876

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Change one image and its description

File: devices/surface/use-system-center-configuration-manager-to-manage-devices-with-semm.md [hunk 1]
```diff
@@ -480,10 +480,10 @@ To add the SEMM Configuration Manager scripts to Configuration Manager as an app
        - Select **Registry** from the **Setting Type** drop-down menu.
        - Select **HKEY_LOCAL_MACHINE** from the **Hive** drop-down menu.
        - Enter **SOFTWARE\Microsoft\Surface\SEMM** in the **Key** field.
-       - Enter **Enabled_Version1000** in the **Value** field.
+       - Enter **CertName** in the **Value** field.
        - Select **String** from the **Data Type** drop-down menu.
        - Select the **This registry setting must satisfy the following rule to indicate the presence of this application** button.
-       - Enter **1** in the **Value** field.
+       - Enter the name of the certificate you entered in the line 58 of the script in the **Value** field.
```
Comment (illfated) [inline devices/surface/use-system-center-configuration-manager-to-manage-devices-with-semm.md]:
```suggestion
       - Enter the name of the certificate you entered in line 58 of the script in the **Value** field.
```
"the line" is only necessary if no other reference is present (in this case, 58)

**Comment by MaratMussabekov** [comment]
Dear @Dansimp ,
could you please have a look at this PR?
Thank you

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5876.

Thank you.

cc: @kenwith

**Pull request merged**